playbooks/roles/caasp4/templates/skuba-terraform-openstack.tfvars.j2

```diff
@@ -29,7 +29,6 @@ packages = [
   "kernel-default",
   "-kernel-default-base",
   "ca-certificates-suse",
-  "patterns-caasp-Node",
```

Comment:
Commit msg does not reflect what is done here. Also I think we don't want to remove the pattern - or do we?

Reply:
we do. Its skuba the one that will install the needed packages on node join, so there is no point in installing those packages already. We should leave it to skuba instead so it can manage the dependencies, versions and whatnot without our intervention :)